sdk/keyvault/azure-security-keyvault-secrets/README.md

```diff
@@ -16,7 +16,7 @@ Maven dependency for the Azure Key Vault Secrets client library. Add it to your
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-security-keyvault-secrets</artifactId>
-    <version>4.2.1</version>
```

Comment:
why was this ? 4.2.1 previously  on Master ?
Isn't master on beta ?

Reply:
The READMEs on master point to the latest stable version, even if they were released from a different branch.

Reply:
okay, its fine for README to point to stable release.
But the master pom should reflect beta. Else this will create confusions for both users and developers in our team when looking at the code.

Reply:
Definitely. The POMs are correctly pointing to the beta versions or Key Vault libraries in master.